linux_os/guide/system/accounts/accounts-restrictions/password_expiration/var_accounts_password_minlen_login_defs.var

```diff
@@ -15,7 +15,7 @@ options:
     15: 15
     6: 6
     8: 8
-    default: 15
+    default: 14
```

Comment:
@cipherboy sorry wasn't clear here
```
14: 14
default: 15
```
Then change the default in the RHEL6 profiles to `14`.

Reply:
@redhatrises All profiles (including RHEL6) already use an explicit value (`var_...=n`). I have reset the default to 15 by dropping this commit. `14` is already a possible value (see [here](https://github.com/OpenSCAP/scap-security-guide/blob/master/linux_os/guide/system/accounts/accounts-restrictions/password_expiration/var_accounts_password_minlen_login_defs.var#L14)).

Reply:
And actually looking that the references.... apparently minlen in now 15 by default anyway.